addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/XBackendServerInformationLeakScanRule.java

```diff
@@ -99,4 +102,9 @@ private String getReference() {
     public Map<String, String> getAlertTags() {
         return ALERT_TAGS;
     }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(createAlert("developer1.webapp.scl3.mozilla.com").build());
```

Comment:
While I'm here, if you navigate to getfirefox.com you should hit a redirect and encounter something like:
`X-Backend-Server: bedrock-5944c575f8-cmlmg.gcp-us-west1` not sure where else it might be seen.

Reply:
> Did you actually encounter this somewhere or is it fictional? If you did then ignore the next comment wink

I used as an example the same evidence that is being used in the automated tests of this class